### ISSUE TITLE
Lets get a team to co-ordinate the Hoodie Twitter account

### ISSUE THREAD

**Issue by Charlotteis** [hoodiehq/editorial#5]
I think Jan/Gregor do the tweets right now and our account isn't very active. Lets get access to a Buffer account for those who want to do some cool tweetz.

**Comment by gr2m** [comment]
:clap:

**Comment by schisepo** [comment]
I'd like to help with the twitter account. I suggest we come up with a social media calendar that determines what gets posted on each weekday e.g. Monday - feature a hood.ie app
Tuesday - Promote an event happening in the ecosystem 
Wednesday - Wacky animal pic

Once we have themes for each day, content can be predetermined and pre-scheduled using something like hootsuite.

**Comment by summerspirit** [comment]
Think you've got great suggestions @schisepo 
I've worked in a couple of organizations where Trello was a good way to set-up 7 columns and then slide in "draft" content for the appropriate day before it goes up on HootSuite.
Then you've got a bunch of things in the pipeline you can slot in when you need to.

**Comment by schisepo** [comment]
@summer I'm not familiar with Trello but it sounds useful ☺ Will look it up.

**Comment by bnb** [comment]
in addition to regular content, engaging with the community in real time is very important. Checking mentions, hashtags, and replying to relevant conversations is one of the most important things to make a social media campaign possible.

Having a few people who can be online throughout the day will greatly help with this. Who it is isn't very important - as long as they have been active contributiors to the editorial team, they should have the chance to take on the social networking roles they want.

**Comment by janl** [comment]
This sounds like a solid plan. Who’s interested in setting up the weekly plan? I’m happy to share Hoodie account credentials any time! Just ping me

**Comment by summerspirit** [comment]
Hi everyone! 
Hope the holidays were good :)

Based on some of the comments here, I have a 2-part solution for tweet planning:
1. Use Trello for planning tweets, collaboration, drafting and discussion
2. Move things from Trello to Tweetdeck once tweets are ready to be scheduled to go live

So, I have:

1. Created a Trello board
  * for planning, drafting and collaborating on tweets
 It's currently listed as "Public", so if you go to this link https://trello.com/b/F7Ofd2x7/weekly-hoodiehq-twitter-planning
  You should be able to sign-up for a Trello account
& then subscribe to the board
  * it is possible to create teams and lock down the board to be private, but we need all the emails for that :)
2. Pre-filled the Trello board
  * with Theme days, based on thoughts here and Twitter traffic stats/info
3. Tried to set-up a Tweetdeck 
https://tweetdeck.twitter.com/
  * I recommend this tool because it's free
  * I would recommend 1 main login/user 
  * Please note a dev friend of mine got it to work but I can't currently login from my location  ¯\_(ツ)_/¯

![screen shot 2016-01-07 at 4 12 33 pm](https://cloud.githubusercontent.com/assets/4708646/12175370/b1721846-b559-11e5-9a76-9af551d6c534.png)

^ me today

---
If this sounds like a good direction,
## Next Steps
- [ ] Sign-up for Trello
- [ ] Subscribe to the Trello board
- [ ] Someone grab responsibility for securing the Tweetdeck
- [ ] Pick a date that we want to start tweeting
- [ ] Add stuff to Trello
- [ ] Move stuff from planning on Trello to "ready to tweet" on Tweetdeck before the date we want to start tweeting

**Comment by janl** [comment]
This is aamaaaaaaazing <3

**Comment by renrutnnej** [comment]
Hi @schisepo @summerspirit @bnb 
– are any you still interested in contributing to the twitter team? I'd like to bump this so we can make some progress on social media stuffs :relaxed:

**Comment by schisepo** [comment]
Hi Jenn, I'm still interested. Am in transition at the moment, new city,
new job. I should be settled in a couple more weeks.

**Comment by renrutnnej** [comment]
Glad to hear it @schisepo, and congrats on your new journey ☺️

**Comment by summerspirit** [comment]
I'm available and interested. Just wanted to move forward as a team :)
Is there a start date we want things in motion?

Cheers,

**Comment by renrutnnej** [comment]
@summerspirit Yay! I'm happy to hear that. As far as a start date, there's nothing keeping us from starting it right now. Is there anything I can help do to get you started? Assuming other contributors will help as they become available of course, so it's not all on you.

**Comment by summerspirit** [comment]
@renrutnnej Well, I just gathered people's suggestions on types of topics we can cover.
Is there content right now that needs communicated now/over the next couple weeks? Like, conferences, workshops, code updates coming...

If you could please point to some titles/events/dates/keywords, that would be super awesome :)
Is there anything that needs to parallel/hook into upcoming newsletters?

**Comment by renrutnnej** [comment]
@summerspirit Do you want to meet sometime and I can help point you to resources and answer your questions? I have availability tomorrow :)

**Comment by summerspirit** [comment]
@renrutnnej That'd be really good!
I have some time tomorrow too. I'm in Dublin time.
My Skype is naomi-hello_world, if that works for you.
Let me know what time would work best :)

**Comment by renrutnnej** [comment]
@summerspirit Are you available at 2pm your time? I can do anytime between then and 5pm (your time).

**Comment by summerspirit** [comment]
@renrutnnej 2 p.m. my time works great! chat with you then :)

**Comment by renrutnnej** [comment]
Hey @summerspirit just wanted to bump this up to see if you're still interested.

**Comment by summerspirit** [comment]
Hey @renrutnnej - it seemed other people took the lead on this and were moving with it, so I was just letting them do that. If you still want any help with it, happy to lend a hand.

**Comment by schisepo** [comment]
@renrutnnej & @summerspirit what was the conclusion on this? Who is currently running the twitter account? they are doing great :smile: Is the process now documented? Do they need help?

**Comment by Charlotteis** [comment]
I think it's still just Gregor/Jan doing occasional tweets?

**Comment by renrutnnej** [comment]
Right now it's a combination of Jan, Gregor and myself (maybe others too, I don't know) so this issue is still one I'd like to see resolved.

**Comment by schisepo** [comment]
There's a checklist put up by @summerspirit. Can we still work with this as a road-map?

**Comment by renrutnnej** [comment]
@schisepo Yes please!! Let me know if you need anything, either by replying to this message or reaching out on Slack. Thank you for volunteering ☺️

**Comment by pawan92** [comment]
im also available to help! =)

**Comment by schisepo** [comment]
Thank you @pawan92 , So far we're revisiting this checklist :point_down: 

## Next Steps
- [ ] Sign-up for Trello
- [ ] Subscribe to the Trello board
- [ ] Someone grab responsibility for securing the Tweetdeck
- [ ] Pick a date that we want to start tweeting
- [ ] Add stuff to Trello
- [ ] Move stuff from planning on Trello to "ready to tweet" on Tweetdeck before the date we want to start tweeting

**Comment by Charlotteis** [comment]
I think that @summerspirit needs to add as to the Trello board so we can add/edit/move stuff around?

**Comment by schisepo** [comment]
You're right @Charlotteis. Is @summerspirit on Slack so that I can reach out?

**Comment by Charlotteis** [comment]
I believe they are [at]nfreeman :)

**Comment by schisepo** [comment]
Hi @everyone I've created a new board on Trello. Please  comment with your email address if you'd like me to add you as a member ☺

**Comment by renrutnnej** [comment]
Add me please! renrutnnej@gmail.com  
  

Jenn Turner  

I &lt;3 great email [jennwrit.es/newsletters](http://jennwrit.es/newsletters
"http://jennwrit.es/newsletters" )

[@jennwrites](https://twitter.com/jennwrites)

  

On Apr 21 2016, at 11:41 am, Sandra Chipendo Chisepo
&lt;notifications@github.com&gt; wrote:  

> Hi [@everyone](https://github.com/everyone) I've created a new board on
Trello. Please comment with your email address if you'd like me to add you as
a member ☺️

>

> —  
You are receiving this because you were mentioned.  
Reply to this email directly or [view it on GitHub](https://github.com/hoodieh
q/editorial/issues/5#issuecomment-212978794)![](https://github.com/notificatio
ns/beacon/ACdpFbDtUKLUt1miQlSzaGQ2p1Otupx4ks5p55qfgaJpZM4Gm9X3.gif)

**Comment by schisepo** [comment]
Here's the invitation link: Join my Trello Board! https://trello.com/invite/b/4GsMspPl/3cb4d83854187880dd22375a7e19c67a/-hoodie

**Comment by renrutnnej** [comment]
@schisepo When you have a process outlined, the next step would be to document it somewhere in this repo and then we can close this issue!

**Comment by schisepo** [comment]
Noted. Will work on it this weekend.

**Comment by schisepo** [comment]
Please review and help flesh out the process. See pull request #89

**Comment by renrutnnej** [comment]
Okay, so while we're reviewing the process, can we clarify who is participating in the twitter team, per the original issue of creating a tweeting team?

**Comment by Charlotteis** [comment]
has any clarification here happened?
